python/runtime/cudaq/spin/py_spin_op.cpp

```diff
@@ -54,16 +54,14 @@ void bindSpinOperator(py::module &mod) {
            "Copy constructor, given another `cudaq.SpinOperator`.")
 
       /// @brief Bind the member functions.
-      .def("get_term_count", &cudaq::spin_op::n_terms,
+      .def("get_raw_data", &cudaq::spin_op::get_raw_data, "")
+      .def("get_term_count", &cudaq::spin_op::num_terms,
            "Return the number of terms in this `SpinOperator`.")
-      .def("get_qubit_count", &cudaq::spin_op::n_qubits,
+      .def("get_qubit_count", &cudaq::spin_op::num_qubits,
```

Comment:
I wrote this before I properly worked out binding class properties -- which our style guide prefers over getters/setters for simple functions like this. I agree with a change to 
```suggestion
      .def_property_readonly("qubit_count", &cudaq::spin_op::num_qubits,
```
This will have to be updated in the docs as well

Reply:
I let you do this in your refactoring activities.